src/mainboard/protectli/vault_jsl/mainboard.c

```diff
@@ -31,6 +31,9 @@ void mainboard_silicon_init_params(FSP_S_CONFIG *params)
 	params->PcieRpLtrEnable[5] = 1;
 	params->PcieRpLtrEnable[6] = 1;
 
+	/* Enable L1 only for WiFi, L0s doesn't work */
+	params->PcieRpAspm[4] = 2;
```

Comment:
```suggestion
	/*
	 * Atheros WiFi cards don't like ASPM L0s, disable it.
	 * ASMedia Switch on V1610 supports only ASPM L1, so no-op */
	if (!CONFIG(BOARD_PROTECTLI_V1610))
		params->PcieRpAspm[4] = FSP_PCH_PCIE_ASPM_L1;
```

And add `#define FSP_PCH_PCIE_ASPM_L1 2` somewhere.

Reply:
@krystian-hebel ping

Reply:
Added: https://github.com/Dasharo/coreboot/compare/20b32be69f8af07d3079c61c1a81cd752c64f156..1c135d6ca4356ea449810e8224955b62b73a070c